dev/bots/test.dart

```diff
@@ -1620,6 +1627,7 @@ Future<void> _runFlutterTest(String workingDirectory, {
   Map<String, String>? environment,
   List<String> tests = const <String>[],
   bool shuffleTests = true,
+  bool fatalLogOutput = true,
```

Comment:
for consistency, this might be better named `fatalLogWarnings`.

Reply:
Good point.  Done.